packages/gestalt/src/PopoverEducational.js

```diff
@@ -12,52 +13,55 @@ import { type Indexable } from './zIndex.js';
 type Size = 'sm' | 'flexible';
 type IdealDirection = 'up' | 'right' | 'down' | 'left';
 type Role = 'dialog' | 'tooltip';
-type PrimaryActionType = {|
-  accessibilityLabel?: string,
-  href?: string,
-  text: string,
-  onClick?: ({|
-    event:
-      | SyntheticMouseEvent<HTMLButtonElement>
-      | SyntheticMouseEvent<HTMLAnchorElement>
-      | SyntheticKeyboardEvent<HTMLAnchorElement>
-      | SyntheticKeyboardEvent<HTMLButtonElement>,
-    dangerouslyDisableOnNavigation: () => void,
-  |}) => void,
-  rel?: 'none' | 'nofollow',
-  target?: null | 'self' | 'blank',
-|};
+type PrimaryActionType =
+  | {|
+      accessibilityLabel?: string,
+      href: string,
+      role: 'link',
+      text: string,
+      onClick?: ({|
+        event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+        dangerouslyDisableOnNavigation: () => void,
+      |}) => void,
+      rel?: 'none' | 'nofollow',
+      target?: null | 'self' | 'blank',
+    |}
+  | {|
+      accessibilityLabel?: string,
+      onClick?: ({|
+        event: SyntheticMouseEvent<HTMLButtonElement> | SyntheticKeyboardEvent<HTMLButtonElement>,
+      |}) => void,
+      role: 'button',
```

Comment:
keep this pattern consistent, alphabetical. Another one had role at the top.

Reply:
DONE